packages/backend/src/shared/utils.ts

```diff
@@ -184,5 +232,10 @@ export async function verifyApiSignature(
   if (!(await canApiSignatureBeProcessed(signature as string, ctx, config)))
     return false
 
-  return verifyApiSignatureDigest(signature as string, ctx.request, config)
+  return verifyApiSignatureDigest(
+    signature as string,
+    ctx.request,
+    config,
+    config.adminApiSecret as string
```

Comment:
```suggestion
    config.adminApiSecret
```
Since it should already be typed

Reply:
nit: but we can pass in either whole of `config` or `adminApiSecret` + `adminApiSignatureVersion` explicitly